DisguiseUnityRenderStream/Runtime/PipelineAbstraction/Presenter.cs

```diff
@@ -158,30 +159,79 @@ static Vector2 CenterUVOffset(Vector2 scale)
     /// mouse coordinates to account for the blit.
     /// </para>
     ///
-    /// <remarks>Assumes that the local screen is the primary display. Modify this class for local multi-monitor specifics.</remarks>
+    /// <remarks>
+    /// Assumes that the local screen is the <see cref="Display.main">main display</see>.
+    /// Modify this class for local multi-monitor specifics.
+    /// Doesn't support HDR display output.
+    /// </remarks>
     /// </summary>
     [ExecuteAlways]
     class Presenter : MonoBehaviour
     {
+        /// <summary>
+        /// The color space of the <see cref="source"/>'s texture.
+        /// </summary>
+        public enum SourceColorSpace
+        {
+            /// <summary>
+            /// Blit directly without any color space conversions.
+            /// </summary>
+            Passthrough = -2,
```

Comment:
Minor: Maybe I'm being a little bit pedantic (sorry), but since this enum is about "The color space of the source's texture", why is one of the constant of the enum about the process we apply to them?

Maybe use "Unspecified" instead?  And in the description say that unspecified textures are blit directly without any color space conversion?